scalableminds/webknossos
# Adds default position, zoom and rotation to datasets

### Mailable description of changes:
 - Adds default position, zoom and rotation to datasets

### Steps to test:
- Find a suitable dataset locally, e.g. `Cortex_knossos`
- Add a `defaultConfiguration` directly in the database:

```json
{"configuration": {"zoom": 6, "layers": {"color": {"color": [255, 255, 255], "contrast": 1, "brightness": 10}}, "fourBit": true, "quality": 0, "position": [0, 0, 0], "datasetName": "", "interpolation": true, "keyboardDelay": 342, "segmentationOpacity": 20}}
```

- Click to view this dataset
- Should be located at (0,0,0) with zoom 6.

### Issues:
- Contributes to https://discuss.webknossos.org/t/enhancements-for-demo-instance/803/4

------
